.env.example

```
@@ -12,6 +12,7 @@ FIREBASE_STORAGE_BUCKET=your_project.appspot.com
 
 # ── Stripe ──────────────────────────────────
 STRIPE_SECRET_KEY=sk_test_...
+STRIPE_PUBLISHABLE_KEY=pk_test_...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_PUBLISHABLE_KEY key should go before the STRIPE_SECRET_KEY key

(UnorderedKey)